problem_builder/swipe.py

```diff
@@ -60,9 +60,9 @@ class SwipeBlock(SubmittingXBlockMixin, StudentViewUserStateMixin, Questionnaire
     )
```

Comment:
We want to remove `QuestionnaireAbstractBlock`, because this swipe component shouldn't have children nor "choices", which is what that mixin adds.

We also want to replace the `question` mixin from `QuestionnaireAbstractBlock` with a simple

```python
    text = String(
        display_name=_("Text"),
        help=_("Text to display on this card. The student must determine if this statement is true or false."),
        scope=Scope.content,
        default="",
        multiline_editor=True,
    )
```

Reply:
Ah, I see.